src/shared/components/query/styles.module.scss

```diff
@@ -61,7 +61,12 @@
   }
 
   :global(.sectionLabel) {
-    width:246px;
+    width:236px;
+  }
+
+  :global(.centeredSectionContent) {
+    margin-left: auto !important;
+    margin-right: auto !important;

```

Comment:
another solution would be to justify the contents to the right instead of centering